tests/end_to_end/test_brute_force/data/test_cases/cases_brute_force.yaml

```diff
@@ -0,0 +1,10 @@
+- name: ssh_brute_force
+  description: Execute brute force attacks using SSH and check generated alerts
+  configuration_parameters: null
+  metadata:
+    rule:
+      id: 5712
+      level: 10
+      description: "sshd: brute force trying to get access to the system. Non existent user."
```

Comment:
```suggestion
    rule.id: 5712
    rule.level: 10
    rule.description: "sshd: brute force trying to get access to the system. Non existent user."
```

Reply:
Done in 85a36ccf7e801c47cce59c4f3fa7a90ec0ac563f